public/js/controllers/groupsSection.controller.js

```diff
@@ -0,0 +1,15 @@
+/*global console*/
```

Comment:
The other one is, this one isn't though; it's not being used right now, but the groups section will be using the data service, and so is expected to report errors via the console in the same way as the conversations section controller